merp_base/views/res_config.xml

```diff
@@ -0,0 +1,116 @@
+                            <div class="o_setting_left_pane">
+                                <field name="module_merp_picking_wave"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_merp_picking_wave"/>
+                                <div class="text-muted">
+                                    Enable Picking Wave
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_left_pane">
+                                <field name="module_merp_outgoing_routing"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_merp_outgoing_routing"/>
+                                <div class="text-muted">
+                                    Add Outgoing Routing strategy options (sort locations)
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_left_pane">
+                                <field name="module_merp_picking_products_skip"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_merp_picking_products_skip"/>
+                                <div class="text-muted">
+                                    Allow smart skip of products in pickings and picking waves
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_left_pane">
+                                <field name="module_merp_instant_move"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_merp_instant_move"/>
+                                <div class="text-muted">
+                                    Allow add more items automatically via Ventor App
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_left_pane">
+                                <field name="module_two_factor_otp_auth"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_two_factor_otp_auth"/>
+                                <div class="text-muted">
+                                    Allows to Use Two Factor Authentication
```

Comment:
This section about 2FA need to be removed from description

Reply:
Deleted